src/main/java/com/amihaiemil/versioneye/RtVersionEye.java

```diff
@@ -92,5 +92,10 @@ public VersionEye trusted() throws IOException {
             this.entry.through(TrustedWire.class)
         );
     }
+
+    @Override
+    public Me meApi() {
```

Comment:
Add it just above the method in the interface and in the classes, with 2-3 lines exception, to catch the signature :D